build.rs

```diff
@@ -0,0 +1,65 @@
+use std::{env, fs::File, io::prelude::*, path::PathBuf};
+
+fn main() {
+    if cfg!(feature = "ld") {
```

Comment:
```suggestion
    if cfg!(feature = "ld") and cfg!(feature = "device-selected" {
```

as discussed in https://github.com/stm32-rs/stm32f3xx-hal/pull/216#discussion_r620266515